### PR TITLE
Add RU synonyms. RU & EN queries for eval.

### PR DESCRIPTION
**Pull request by bbfsdev**

### CODE REVIEW

File: search/variables/convention_location.variable [hunk 1]
```diff
@@ -168,86 +168,86 @@ he,United States|New Jersey => ניו גרזי
...
+ru,Romania|Iasi => Яссы
+ru,Russia|Krasnoyarsk => Красноярск
+ru,Russia|Moscow => Москва
+ru,Russia|Novosibirsk => Новосибирск
+ru,Russia|Saint Petersburg => Санкт-Петербург
+ru,Russia|Sochi => Сочи
+ru,Spain|Madrid => Мадрид
+ru,Sweden|Stockholm => Стокгольм
+ru,Switzerland|Berne => Берн
+ru,Turke|Antalya => Анталия
```
Comment (bbfsdev) [inline search/variables/convention_location.variable]: Should be Turkey

**Comment by YuriHechter** [comment]
[Checking Russian synonyms](https://trello.com/c/9W70HMEg/47-checking-russian-synonyms)

**Comment by YuriHechter** [comment]
[Checking Russian synonyms](https://trello.com/c/9W70HMEg/47-checking-russian-synonyms)

**Comment by YuriHechter** [comment]
[Checking Russian synonyms](https://trello.com/c/9W70HMEg/47-checking-russian-synonyms)

File: search/variables/convention_location.variable (hunk 1 above)
Comment (YuriHechter) [reply search/variables/convention_location.variable]: fixed

**Pull request merged**